src/Samples/Builder.Console/Builder.Console.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1;net6.0</TargetFrameworks>
```

Comment:
Why was `netcoreapp3.1` kept?

Reply:
that way, the project is compatible to be used in dotnet core 3.1 projects too